### DISCUSSION TITLE
Reopening the conversation: TypeError: Cannot read properties of undefined (reading 'startsWith')

### DISCUSSION THREAD

**Discussion by HMKahn006** [FlowiseAI/Flowise#2183] [Q&A]
Hi All, 
I'm trying to overcome this error in a QA chatbot. The goal was to explore the possibility of an AI tutor of sorts which is based on a given text/curriculum. But so far I cannot get past this _TypeError_ issue. I haven't been able to find much information on how to fix this issue within flowise. The other thread that mentions the same error is currently unanswered. Posting my flow here in case it's caused by my parameters. Thanks for any help!

<img width="1175" alt="Screen Shot 2024-04-14 at 12 20 09 PM" src="https://github.com/FlowiseAI/Flowise/assets/166341627/aee2315d-67c9-4fdd-b138-004c924d86a5">

**Comment by HenryHengZJ** [comment]
Have you upserted the data to Pinecone?
